rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1741,6 +1741,18 @@ String loginWithCredential(@MultipartForm LoginForm multipart)
     @Produces("application/json")
     List<String> globalspaceURIs(@HeaderParam("sessionid") String sessionId) throws RestException;
 
+    @GET
+    @Path("globalspace/{pathname}/exist")
```

Comment:
the dedicated dataspace rest api is part of the RestDataspace (/data). Methods which are inside SchedulerRestInterface are legacy.